tests/tools/test_utils.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
Which file is this unit test for?
For all the unit test of path/to/filename.py, the name should be tests/path/to/test_filename.py

Reply:
Let me change back to this style. I used this before , but rui want to add more tests into this file. But it looks better to separate the test.